package.json

```diff
@@ -53,9 +54,15 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint && yarn test"
+      "pre-commit": "lint-staged",
+      "post-commit": "git update-index --again",
+      "pre-push": "yarn test"
     }
   },
+  "lint-staged": {
+    "*.js": ["prettier --write", "eslint --fix --max-warnings 0", "git add"],
+    "*.scss": ["stylelint --fix --syntax=scss", "git add"]
```

Comment:
@olslash Thoughts on `stylelint --fix`?

Reply:
this one is ok i think

Reply:
I'm thinking maybe we don't want it though just to stay consistent if we're not gonna use `eslint --fix`.  Encourages a consistent culture of "don't commit errors".

Reply:
sure